### ISSUE TITLE
ECO-pytorch Performance on Kinetics validation set

### ISSUE THREAD

**Issue by arunos728** [mzolfaghari/ECO-efficient-video-understanding#37]
Could you report ECO-pytorch Performance on Kinetics validation set (ECO-Lite)?

I evaluated Kinetics-pretrained ECO-Lite model from ECO-pytorch on Kinetics validation set,
only got 62.1% which is not as good as 64.4% in paper. (ECO-Lite, number of segment = 16)

**Comment by mzolfaghari** [comment]
@arunos728 We didn't evaluate ECO-Lite pytorch version on Kinetics. But one of the problems, I think is related to the dropout location. In the Caffe version we have dropout after global_pool layer but in the pytorch version, it's in the final layer.

In the coming week, we will release the ECO-full pytorch version pretrained on Kinetics. This version will not have a problem and results are similar to the Caffe version.

**Comment by ntuyt** [comment]
Hi, could you release the ECO-full pytorch version pretrained on Kinetics? Thanks so much

**Comment by mzolfaghari** [comment]
Sure. We'll release the ECO-full pre-trained models next week.
